### ISSUE TITLE
Result-image-size option for resizing only larger pictures

### ISSUE THREAD

**Issue by laura-chesches** [CrackerakiUA/ui-cropper#69]
Hello. 
Is there any possibility to use result-image-size, or to create a new functionality result-image-min-size that resizes only the larger files / cropped areas?
I have to make sure that there aren't any files or cropped images that have a resolution smaller than 1024 and also I have to resize the bigger Images to 1024. 
This new functionality would be very helpful.
Thank you

**Comment by CrackerakiUA** [comment]
Agree, you are welcome to contribute, sounds useful but i don't currently have time to implement it.
If you decide to do so, feel free to ask any question here.
